server_admin/topics/clients/saml/proc-entity-descriptors.adoc

```diff
@@ -0,0 +1,23 @@
+[id="proc-entity-descriptors_{context}"]
+
+==== Using an entity descriptor to create a client
+
+An alternative to registering a SAML 2.0 client manually, you can import the client using a standard SAML Entity Descriptor XML file.
+
+There is an `Import` option on the Add Client page.
```

Comment:
I generally avoid "There is" because it makes no sense grammatically.  For example, is "there" the subject of the sentence?
I would say "The Add Client page includes an `Import` option.